### DISCUSSION TITLE
How to switch from JDownloader2 to pyLoad? / How to import DLCs etc from JDownloader2?

### DISCUSSION THREAD

**Discussion by ihasaquesion** [pyload/pyload#3982] [Help]
### Question
I'd like to switch from JDownloader2 to pyLoad.
This is because [I could not get JDownloader2 (opening captchas in the browser) to work properly with firejail sandboxing (and because it kept having problems starting up)](https://github.com/netblue30/firejail/issues/3579), I prefer python over Java and because it looks like JDownloader2 isn't fully open source.

I'd like to keep DLCs and as many other things as possible (maybe some configs and the list of passwords for unpacking compressed files) so that pyLoad picks up downloads where they left in JDownloader2.

What's the best way to do this? How to most easily import as much as possible from JD2 and are there any plans for improving this import further?

### Additional references

**Comment by GammaC0de** [comment]
To send a DLC file to pyLoad, you have to upload it to pyload
![image](https://user-images.githubusercontent.com/11036551/101288489-a509c080-37ff-11eb-8335-d0b59d2c20d2.png)

**Comment by ihasaquesion** [comment]
Bot how would the full procedure of switching from JDownloader2 to pyLoad look like?

Going to folder x where DLC files of JDownloader2 are stored, adding all the DLC files to pyLoad individually, then going to folder y where the partially downloaded files from JDownloader2 are stored and moving them into download directory z of pyLoad?

Is there a way to add all DLCs in a folder, or which DLCs are currently added in JD2 (I think not all DLCs in the folder might be added in JD2) and can other configurations like the decompression passwords be imported too?

**Comment by GammaC0de** [comment]
Just install pyLoad and start download.

If you want to move all downloads you have started with jdownloader, you would have to do manually, there is no some kind of "automatic import" procedure.

**Comment by ihasaquesion** [comment]
Alright, thanks - that's not good news if you wish for people to switch to pyLoad though. If you'd like more people to use pyLoad I think it would be good to make the switch easier/better supported and more convenient (at least a simple guide would probably help). An "import" would be a very useful feature - I'll create a new issue for it, close this issue and link to here from there if that's okay.
